zotero_master/translators/TyndaleBulletin.js

```diff
@@ -0,0 +1,371 @@
+{
+	"translatorID": "04938f7b-4cda-4efa-bbe8-efc2fd6decb2",
+	"label": "TyndaleBulletin",
+	"creator": "Helena Nebel",
+	"target": "tyndalebulletin.org\\/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 99,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-01-27 15:10:50"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	AEA Web translator Copyright © 2014 Sebastian Karcher 
```

Comment:
Bitte ändern in: `Copyright © 2022 Universitätsbibliothek Tübingen.  All rights reserved.`

Reply:
Nach merge ändere ich das noch selbst.